src/base/bittorrent/tracker.cpp

```diff
@@ -244,6 +244,9 @@ Http::Response Tracker::processRequest(const Http::Request &request, const Http:
             throw NotFoundHTTPError();
     }
     catch (const HTTPError &error) {
+        LogMsg(tr("Embedded Tracker: HTTP request error: %1 %2")
+                .arg(QString::number(error.statusCode()), error.statusText())
+            , Log::WARNING);
```

Comment:
Log::WARNING is misused. Any error from client isn't relevant to qbt normal operations.

Reply:
fixed